packages/web-components/src/tabs/tabs.ts

```diff
@@ -383,6 +383,21 @@ export class Tabs extends BaseTabs {
   @attr
   public size?: TabsSize;
 
+  /**
+   * Iterates over each tab element and sets its `disabled` attribute based on the `disabled` property of the `Tabs` component.
+   * If the `disabled` property is `true`, the `disabled` attribute of the tab element will be set to `'true'`; otherwise, it will be set to `'false'`.
+   */
+  protected setTabs(): void {
+    super.setTabs();
+
+    this.tabs.forEach((tab: HTMLElement) => {
+      // Only set the 'disabled' attribute if it's not already set on the individual tab
+      if (!tab.hasAttribute('aria-disabled')) {
+        tab.setAttribute('aria-disabled', this.disabled ? 'true' : 'false');
+      }
+    });
+  }
```

Comment:
This is completely in the users control, is the intent here convenience?

Reply:
I'd say this is largely convenience. It may be unclear that the engineer needs to set a disabled state on each individual tab after they set the disabled state on the entire group.